fibjs/src/websocket/WebSocket.cpp

```diff
@@ -15,13 +15,14 @@
 #include <mbedtls/mbedtls/sha1.h>
 #include "encoding.h"
 #include "MemoryStream.h"
+#include "HttpClient.h"
 #include <stdlib.h>
 
 namespace fibjs {
 
 DECLARE_MODULE(ws);
 
-result_t http_request(exlib::string method, exlib::string url,
+result_t http_request2(obj_ptr<HttpClient_base> httpClient, exlib::string method, exlib::string url,
```

Comment:
```suggestion
result_t http_request2(obj_ptr<HttpClient_base> httpClient, exlib::string method, exlib::string url,
```
obj_ptr 作为参数时，只需要传递指针，比如这里，可以直接如下定义：
```suggestion
result_t http_request2(HttpClient_base* httpClient, exlib::string method, exlib::string url,
```

Reply:
以下相同。